akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -270,6 +297,26 @@ class ClusterClient(settings: ClusterClientSettings) extends Actor with Stash wi
     else if (contacts.size == 1) (initialContactsSel ++ contacts) foreach { _ ! GetContacts }
     else contacts foreach { _ ! GetContacts }
   }
+
+  def buffer(msg: Any): Unit =
```

Comment:
For my use-cases that would typically be better. But some way of plugging into that would be nice.

On the other hand, if you have retry implemented on the sender, the order would be less effected if you continued to drop the new messages. (there is not a strict ordering guarantee anyway, but dropping the oldest would cause them to be resent, and the ordering would be even more off)